components/edit-collective/sections/Host.js

```diff
@@ -88,6 +88,20 @@ class Host extends React.Component {
     }
   }
 
+  renderLegalNameSetInfoMessage(collective) {
+    return (
+      <MessageBox type="info" fontSize="13px" withIcon>
+        <FormattedMessage
+          id="collective.edit.host.legalName.info"
+          defaultMessage="Please set the legal name of the host in the Info section under {settingsLink}. This is required if your legal name is different than your display name for tax and accounting purposes."
+          values={{
+            settingsLink: <Link href={`/${collective.host?.slug}/edit`}>Settings</Link>,
+          }}
```

Comment:
The word `Settings` won't be translated here, which will look a bit strange :wink: 

> Veuillez définir le nom légal de l'hôte dans la section info sous settings. Ceci est nécessaire si votre nom légal est différent de votre nom d'affichage pour des raisons fiscales et comptables.

As per https://docs.opencollective.com/help/contributing/development/translations#translate-links-inline, the simplest approach here is `getI18nLink`:
```suggestion
          defaultMessage="Please set the legal name of the host in the Info section under <SettingsLink>your settings</SettingsLink>. This is required if your legal name is different than your display name for tax and accounting purposes."
          values={{
            SettingsLink: getI18nLink({ href: `/${collective.host?.slug}/edit` })
          }}
```

Reply:
@Betree : Oh, that's a good point. I didn't think about this one. Have to keep this in mind. 🤔